custom-namespaces.yaml

```
@@ -506,6 +506,19 @@ revenue_private:
       type: table_view
       tables:
         - table: mozdata.revenue_cat3_analysis.budget_complete_2022
+ads:
+  glean_app: false
+  connection: bigquery-oauth
+  pretty_name: Ads
+  owners:
+    - cmorales@mozilla.com
+    - sbetancourt@mozilla.com
+  spoke: looker-spoke-private
+  views:
+    consolidated_ad_metrics:
+      type: table_view
+      tables:
+        - table: mozdata.ads.consolidated_ad_metrics
```

We should also pull in the Pacific Time table and the hourly table

```suggestion
        - table: mozdata.ads.consolidated_ad_metrics
        - table: mozdata.ads.consolidated_ad_metrics_daily_pt
        - table: mozdata.ads.consolidated_ad_metrics_hourly
```

---

@curtismorales : Makes sense. Do we need to add any conditions or filters to separate these three out? Looking at https://github.com/mozilla/lookml-generator/blob/5c771f8c49f39788a597f919f678345904f6abd2/custom-namespaces.yaml#L229

---

Yeah, let's do this for now. We can optimize later. Thank you.